vignettes/tool-calling.Rmd

```diff
@@ -65,6 +65,8 @@ get_current_time <- function(tz = "UTC") {
 }
 ```
 
+Note that we've gone through the trouble of creating [roxygen2 comments](https://roxygen2.r-lib.org/). This is a very important step that will help the model use your tool correctly!
```

Comment:
> This is a very important step that will help the model use your tool correctly!

This sentence (and moving it up in the doc) makes it sound like `tool()` will automatically use the roxygen2 docs or that it's required to create a tool.

I think you should:

1. Move this paragraph to just before "This is a fair amount of code to write..." where you introduce `create_tool_def()`
2. Soften the language a bit to say that writing the roxygen2 docs can save time (rather than being an important step for correctness)